tools/please_go/generate/generate.go

```diff
@@ -231,6 +231,9 @@ func (g *Generate) writeConfig() error {
 	fmt.Fprintln(file, "[Plugin \"go\"]")
 	fmt.Fprintln(file, "Target=@//plugins:go")
 	fmt.Fprintf(file, "ImportPath=%s\n", g.moduleName)
+	for t := range g.buildContext.BuildTags {
```

Comment:
Is this actually redundant because its already on the build context?

Reply:
Sort of. I think it's probably best to do it here too.